jenkins/images.sh

```diff
@@ -32,6 +32,7 @@ fi
 bin/cork update \
     --create --downgrade-replace --verify --verify-signature --verbose \
     --sdk-url-path "${SDK_URL_PATH}" \
+    --json-key "${GOOGLE_APPLICATION_CREDENTIALS}" \
```

Comment:
Below the following is used:
```
sudo ln -f "${GS_DEVEL_CREDS}" chroot/etc/portage/gangue.json
...
--json-key=/etc/portage/gangue.json $verify_key
```
So should it be `GS_DEVEL CREDS`?

Reply:
right, like with `jenkins/vm.sh` - I missed this one, thanks !